### PR TITLE
Add frgx finance token

### PR DESCRIPTION
**Pull request by Valeriy0**

### CODE REVIEW

**Comment by Valeriy0** [comment]
https://www.coingecko.com/en/coins/frgx-finance
https://coinmarketcap.com/currencies/elanausd/

I will duplicate the links to сmс and coingecko.
We also have more than 28k+ holders.

**Comment by cryptomanz1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by Valeriy0** [comment]
• Many of our users use Trust wallet, so we really want to provide them with complete information about the token in your wallet. The automatic transmission token is an ecosystem token, which will later be presented in all our products
https://frgx.finance/docs/frgx_overview.pdf

• The token is only bought by our community and other market participants. We do not conduct any airdrops

• Initially, the token was placed using the fair launch system (100% of the tokens were initially available for purchase) 
• At the moment, you can only buy our token for pancake, but soon we will be added to one of the tier 2-3 exchanges
https://pancakeswap.finance/swap?inputCurrency=BNB&outputCurrency=0xC703dA39aE3B9Db67C207c7Bad8100E1AfdC0F9c

• audit - https://frgx.finance/docs/audit_FRGX_approved.pdf

**Comment by Valeriy0** [comment]
@Cryptocool1 @cryptomanz1 Hello everyone, tell me the logo in the wallet, we saw it, when will the price and schedule be visible?

File: blockchains/smartchain/assets/0xC703dA39aE3B9Db67C207c7Bad8100E1AfdC0F9c/info.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (mostafalarki1368mos) [inline blockchains/smartchain/assets/0xC703dA39aE3B9Db67C207c7Bad8100E1AfdC0F9c/info.json:1]: > # - - [ ] ~~@Cryptocool1 @cryptomanz1 @Valeriy0 @vcoolish @tommid @philiparthurmoore @fgallaire @gaia ~~

Comment (mostafalarki1368mos) [inline blockchains/smartchain/assets/0xC703dA39aE3B9Db67C207c7Bad8100E1AfdC0F9c/info.json:1]: @cryptomanz1 @covain @Cryptocool1 @curiouschonk @curiouschonk @rfunduk @hems @chuckbergeron @cloudhead @tommid @zmanian @gaia []()

Comment (mostafalarki1368mos) [inline blockchains/smartchain/assets/0xC703dA39aE3B9Db67C207c7Bad8100E1AfdC0F9c/info.json:1]: @cryptomanz1 @covain @Cryptocool1 @curiouschonk @curiouschonk @rfunduk @hems @chuckbergeron @cloudhead @tommid - [ ] @zmanian @gaia []()mehersalarki2598@gmail.com.

**Review by mostafalarki1368mos — COMMENTED** [review]
- [ ]

**Comment by gaia** [comment]
@Valeriy0 please block this user from commenting with spam https://github.com/trustwallet/assets/pull/27164#discussion_r1454005700

**Comment by Valeriy0** [comment]
@Cryptocool1 @cryptomanz1 Hello everyone, we have already seen the logo in the wallet, but when will the price and schedule be visible?

**Pull request merged**